pkg/client/README.md

```diff
@@ -0,0 +1,132 @@
+# William
+
+This library aims to prevent writing to much code using william on your application.
+
+## Creating a service
+```go
+wi := william.New("URL OF WILLIAM", "ServiceName")
+```
+You can set the HTTP Client that will made the request also the tracing if anyone. (if not set nil)
+```go
+wi.SetClient(
+    &http.Client{
+        Timeout: time.Second * 2,
+    },
+    nil,
+)
+```
+
+If your service will use any internal function of william like a service, Ej: adding permission directly from your service and not from the Hub you need to set a KeyPair to auth.
+
+```go
+wi.SetKey(
```

Comment:
Does the client provide the feature to create `Service Accounts` in Will.IAM? (Even if it does, Will.IAM does not return the KeyPair on account creation). It would be nice to note here that first you need a Service Account and the KeyPair data to use the client. WDYT?

Reply:
Yes, the idea is to add this method, now, only support ListPermission, that will list the services account with permission that the Key Pair of the service is the owner.